docs/index.md

```diff
@@ -9,12 +9,18 @@ description: |-
 
 The HCP provider provides resources to manage [HashiCorp Cloud Platform](https://cloud.hashicorp.com/) (HCP) resources.
 
+~> **Known Issue** There is an issue with v0.4.0 of the HCP Provider in which existing Consul clusters that do not specify size will be force-recreated on the next Apply. Please upgrade to the patch v0.4.1 to avoid this issue.
+
 ## Authenticating with HCP
 
 The HCP provider supports authentication via a Client ID and a Client Secret. The [authentication guide](guides/auth.md) describes how to obtain client credentials.
 
 ## Example Usage
 
+Most HCP resources have a dependency on the Hashicorp Virtual Network (HVN) resource. 
```

Comment:
Nit: IMO this is so fundamental to Terraform usage that it would be extraneous information for the average user (I'm not aware of any other providers that include a note like this), and for Terraform beginners, we probably need to be directing them to more comprehensive guides to be brought up to speed